docs/documentation/make-first-prototype/branching.md

```diff
@@ -2,15 +2,15 @@
 
 Our first question asks the user how many balls they can juggle. We’re going to send them to an ‘ineligible’ page if they can only juggle 2 balls or less. Sending users to different pages based on their input is called branching.
 
-To do this, we’re going to need an ‘ineligible.html’ page, and some logic to decide when to send users there.
+To do this, we’re going to need an `ineligible.html` page, and some logic to decide when to send users there.
 
 This kind of logic goes in a file called `app/routes.js`, which is written in JavaScript. Routes tell the kit what to do when the user goes to specific pages.
 
 The route takes the answer the user gave to the first question and either sends them to the next question or the ineligible page, depending on their answer.
 
 ## Create an ineligible page
 
-1. Make an `ineligible.html` page by copying `template-content-page.html`.
+1. Make an `ineligible.html` page by copying `layout.html` from `docs/views/templates` to `app/views`.
```

Comment:
I think its `content.html` not `layout.html`:
https://github.com/alphagov/govuk-prototype-kit/blob/main/docs/views/templates/content.html

Reply:
Oh yeah I knew that 🤦